src/collection/resources/company.js

```diff
@@ -12,14 +12,6 @@ class Company extends Component {
       type: 'get', action: 'properties', urlParams, ResConstructor: null,
     }, done);
   }
-
-  get({ urlParams, json = false } = {}, done) {
```

Comment:
we don't need this anymore?

Reply:
Not for the core-sdk, users shouldn't get the whole company.